docs/release_process.rst

```diff
@@ -0,0 +1,175 @@
+=====================
+LORE Release Workflow
+=====================
+
+Quick Start
+-----------
+
+prerequisites:
+  - ``git-release-notes`` - https://www.npmjs.com/package/git-release-notes
+  - ``git-flow`` - https://github.com/nvie/gitflow
+  - You've configured Git-Flow to use LORE branch naming scheme.
+
+These instructions presume you're releasing 0.10.0.
+
+.. code-block:: bash
+
+    git fetch
+    git checkout master
+    git pull
+    git flow release start 0.10.0
+    vi lore/settings.py # change version number
+    git-release-notes v0.9.0..master util/release_notes_rst.ejs
+    vi RELEASE.rst # paste results of prior command at top of file, correcting the release number
+    git add RELEASE.rst lore/settings.py
+    git commit -m "Release 0.10.0"
+    # verify that the Travis-ci build succeeds
```

Comment:
There are some steps missing here about pushing the rc branch and merging it into the release-candidate branch